### DISCUSSION TITLE
Setup Wizard cannot find python in the system.

### DISCUSSION THREAD

**Discussion by Baizkunthaz** [learningequality/kolibri#11648] [Q&A]
![Screenshot (5)](https://github.com/learningequality/kolibri/assets/144135126/c361dc76-f28b-4719-97fa-98a456ae7c76)

i m using windows 11 and also i have python successfully installed on my system but still its showing like this, I have tried every possible way to correct it but I couldn't find any way, can anyone suggest me on how to resolve it.

**Comment by bjester** [comment]
Hi @Baizkunthaz, there could be more than one reason our installer shows this modal. I thought it might be helpful to share that the default path it checks for python/pip is: `\Python38\Scripts\pip.exe` 
src: https://github.com/learningequality/kolibri-installer-windows/blob/55734e966d1d1ad0ee7e0b8c4cac163fb7e35b74/src/installer-source/KolibriSetupScript.iss#L466

**Comment by Baizkunthaz** [comment]
So, what could I do to overcome this issue, sir? Can you please explain to me briefly how could I execute this command on Windows?

**Comment by radinamatic** [comment]
@Baizkunthaz another option would be to set up a Linux virtual machine to use as a development environment instead of your host Windows system. You can use the free platform called **Virtual Box** to install Ubuntu and avoid various issues of development on Windows. [Check this quick start guide on how to proceed](https://docs.google.com/document/d/195sDx9TxaXtGP5_Ao1NIsMq_eSDzeveWi84jO5p0yNE/edit?usp=sharing).

Running a virtual machine may put some strain on your computer as you need to allocate enough resources for the development to be efficient, but any relatively modern device should be able to take it. I've been using a virtual machine to run the Kolibri in development on Ubuntu for several years now, and I usually allocate 8GB of RAM and 2 CPU cores, but I do remember my beginnings with our previous platform KA Lite on my old laptop, where my virtual machine had half that in terms of the resources. It was slow, but it did run 😉

**Reply by Baizkunthaz** [reply]
Loved to hear this; thanks a lot mam for the guidance. I will install Ubuntu, and I hope that I'll be able to make some quality contribution towards Kolibri; the only thing I need is yours support.

**Comment by jredrejo** [comment]
@Baizkunthaz  in any case, if you already have a previous Python version that's 3.6+ and can't continue with the installation, that's an issue. In order to reproduce your problem, can you tell what python version do you have, how did you install it and the path to the installed python binaries in your system?

**Reply by Baizkunthaz** [reply]
I have Python version 3.9.4. I had it previously and downloaded it from the official Python website.

**Comment by MisRob** [comment]
Hi @Baizkunthaz, I'm sorry we missed this - the notifications for sub-threads are troublesome. Will close since it's been a long time, but please post again if you had any issues.
